PHPCI/Helper/Lang.php

```diff
@@ -168,4 +168,20 @@ protected static function loadAvailableLanguages()
             }
         }
     }
+
+    /**
+     * Create a time tag for localization.
```

Comment:
https://github.com/Block8/PHPCI/wiki/Contributing-to-PHPCI#coding-standards

> ... If you're modifying a file as part of your change which does not comply with the above, please make the necessary changes to bring it into compliance prior to submitting the pull request.

[PhpDocumentor](http://www.phpdoc.org/docs/latest/getting-started/your-first-set-of-documentation.html#what-does-a-docblock-look-like) references.

Yes, please :smile: